src/components/ContentNav/SubItem/index.tsx

```diff
@@ -22,6 +22,8 @@ export interface SubItemProps {
  * @param {(e: React.MouseEvent<HTMLLIElement>, n: string) => void} [activate]
```

Comment:
"HTMLLIElement" = typo?

Reply:
This one is correct I believe. It is referring to an html li element.